packages/worker/src/worker-options.ts

```diff
@@ -185,17 +187,31 @@ export interface WorkerOptions {
    */
   dataConverter?: DataConverter;
 
+  /**
+   * Provide a custom {@link WorkerTuner}.
+   *
+   * Mutually exclusive with the {@link maxConcurrentWorkflowTaskExecutions}, {@link
+   * maxConcurrentActivityTaskExecutions}, and {@link maxConcurrentLocalActivityExecutions} options.
+   */
+  tuner?: WorkerTuner;
+
   /**
    * Maximum number of Activity tasks to execute concurrently.
    * Adjust this to improve Worker resource consumption.
-   * @default 100
+   *
+   * Mutually exclusive with the {@link tuner} option.
+   *
+   * @default 100 if no {@link tuner} is set
```

Comment:
Just a note that once `WorkerTuner` reaches GA, we can consider deprecating these options, promoting explicitly using a Fixed Size tuner instead.